CompanyEmployees.Application/Validators/CheckInAttendanceCommandValidator.cs

```diff
@@ -0,0 +1,37 @@
+using CompanyEmployees.Application.Commands;
+using FluentValidation;
+using FluentValidation.Results;
+
+namespace CompanyEmployees.Application.Validators;
+
+public sealed class CheckInAttendanceCommandValidator : AbstractValidator<CheckInAttendanceCommand>
+{
+    public CheckInAttendanceCommandValidator()
+    {
+        RuleFor(a => a.Attendance.EmployeeId)
+            .NotEmpty()
+            .WithMessage("Employee ID is required.");
+
+        RuleFor(a => a.Attendance.CheckInTime)
+            .NotEmpty()
```

Comment:
Consider chaining `.WithMessage(...)` after `.NotEmpty()` for improved readability, consistent with other rules in this class.

**Suggested change**
```suggestion
RuleFor(a => a.Attendance.CheckInTime)
    .NotEmpty().WithMessage("Check-in time is required.");
```